play/src/front/Chat/Connection/Matrix/CustomMatrixLogger.ts

```diff
@@ -0,0 +1,42 @@
+import Debug from "debug";
+import { Logger } from "matrix-js-sdk/lib/logger";
+
+const traceLogger = Debug("trace");
+const debugLogger = Debug("debug");
+const infoLogger = Debug("info");
+const warnLogger = Debug("warn");
+const errorLogger = Debug("error");
```

Comment:
Let's prefix those debuggers with the "matrix:" prefix.

```suggestion
const traceLogger = Debug("matrix:trace");
const debugLogger = Debug("matrix:debug");
const infoLogger = Debug("matrix:info");
const warnLogger = Debug("matrix:warn");
const errorLogger = Debug("matrix:error");
```